configs/config.json.example

```diff
@@ -13,6 +13,9 @@
     "test": false,
     "location_cache": true,
     "distance_unit": "km",
+    "pokestop_cooldown": "300",
```

Comment:
Pokestop cooldown is always 300 seconds (5 minutes). Why would we let users customise this? It doesn't make sense.